app/models/intake.rb

```diff
@@ -454,6 +454,16 @@ def year_before_most_recent_filing_year
     (most_recent_filing_year.to_i - 1).to_s if most_recent_filing_year.present?
   end
 
+  def marital_statuses
+    [
+        ("Married" if married_yes?),
+        ("Lived with spouse" if lived_with_spouse_yes?),
+        ("Legally separated #{separated_year}" if separated_yes?),
+        ("Divorced #{divorced_year}" if divorced_yes?),
+        ("Widowed #{widowed_year}" if widowed_yes?)
+    ].compact.presence || ["Single"]
```

Comment:
This is really clear, I like how readable this is. 

Also, I would have expected these to be translation strings. Is the a reason we wouldn't want to?

Reply:
We just realized they should be!! Thanks :D